app/controllers/application_controller.rb

```diff
@@ -2,4 +2,6 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  include SessionsHelper
```

Comment:
Rather than including all methods from your session helper here, define the methods you want to use in the controller here, and then tell Rails you want them to be helper methods, too. E.g.

```ruby
class ApplicationController < ActionController::Base
    helper_method :current_user, :logged_in?

    def current_user
      @current_user ||= User.find_by_id(session[:user])
    end

     def logged_in?
       current_user != nil
     end
  end
```

See [this API page](http://apidock.com/rails/ActionController/Helpers/ClassMethods/helper_method) for more.